projects/kubernetes/image-cache/Dockerfile

```diff
@@ -1,5 +1,22 @@
-FROM docker:17.06.0-ce-dind
-ADD . /images
+FROM docker:17.06.0-ce-dind AS docker
```

Comment:
as you just want the client you can just use `docker:17.06.0` (which `dind` inherits from)